### ISSUE TITLE
Let each entry choose which sites it should be saved to - Issue with Reordering of enabled pages

### ISSUE THREAD

**Issue by bkrsmanovic** [craftcms/cms#9843]
### Description
Entries ordering are connected between sites once you enable pages for other sites. It makes it really hard to organize navigations and cause other issues if you are to use order of pages.

### Steps to reproduce
1. When you have multisite, and you have pages on Site 1 (example):
![Screenshot from 2021-09-16 11-17-21](https://user-images.githubusercontent.com/15169280/133585888-5de8cc40-e5ec-4e49-b82d-94713c6ba71e.png)

2. Then when you go to each page and enable them for Site 2 example):
![Screenshot from 2021-09-16 11-12-33](https://user-images.githubusercontent.com/15169280/133585157-6530673b-f094-472a-88f3-a5e594d02dea.png)
-- Renamed them on Site 2 to make it distinquish:
![Screenshot from 2021-09-16 11-17-49](https://user-images.githubusercontent.com/15169280/133585972-ba805f78-3f01-467b-897f-8d3840a22a48.png)

3. When for example, admin for Site 2 decides to change order of pages on Site 2 (Subentry 2 above Subentry 1):
![Screenshot from 2021-09-16 11-18-45](https://user-images.githubusercontent.com/15169280/133586122-4c7dfabd-a6af-4a8a-9d08-1713222aadbe.png)

Order changes for main site as well, despite Site 2 admin not having access to Site 1. This happens even if **global admin** does change:
![Screenshot from 2021-09-16 11-20-38](https://user-images.githubusercontent.com/15169280/133586372-ace17379-b392-43ab-9971-061e619f5abf.png)

TLDR:
I think ordering of pages should be kept on site-level. Or, there should be a checkbox or something on each page, that can 'Separate from original page', or something like that. Not even global admin should affect ordering on 1 site, and pass it onto other sites.
This prevents subsequent sites from having their own page orders, without affecting other sites.
This is also connected on ALL sites that enable same entries, not just previous-enabled sites.

### Additional info

- Craft version: Craft CMS 3.7.12
- PHP version: 8
- Database driver & version: MariaDB
- Plugins & versions: /

**Comment by brandonkelly** [comment]
This is working as expected… there is only one structure per Structure section, and entries have a single place in them, across all their sites.

Maybe in your case you should not allow entries to belong to multiple sites, or create multiple Structure sections (one for each site).
